Remove accidently used `wiremock.com.google.common.base.Objects`

## Situation
Our develop branch currently fails with

## Wanted
```
> Task :sechub-scan-product-pds:compileJava
/home/runner/work/sechub/sechub/sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertAutoCleanupInspections.java:8: error: package wiremock.com.google.common.base does not exist
import wiremock.com.google.common.base.Objects;

> Task :sechub-integrationtest:compileJava FAILED
132 actionable tasks: 130 executed, 2 up-to-date
                                      ^
/home/runner/work/sechub/sechub/sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertAutoCleanupInspections.java:140: error: cannot find symbol
                if (!Objects.equal(variant, count.variant)) {
                     ^
  symbol:   variable Objects
  location: class com.mercedesbenz.sechub.integrationtest.api.AssertAutoCleanupInspections.AssertAutoCleanupDeleteCountFoundAction
/home/runner/work/sechub/sechub/sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertAutoCleanupInspections.java:143: error: cannot find symbol
                if (!Objects.equal(className, count.className)) {
                     ^
  symbol:   variable Objects
  location: class com.mercedesbenz.sechub.integrationtest.api.AssertAutoCleanupInspections.AssertAutoCleanupDeleteCountFoundAction
3 error
```

## Solution

Replace
```
import wiremock.com.google.common.base.Objects;
```

with `java.util.Objects`

--- a/sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertAutoCleanupInspections.java
+++ b/sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertAutoCleanupInspections.java
@@ -2,10 +2,9 @@ package com.mercedesbenz.sechub.integrationtest.api;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import org.junit.Assert;
-
-import wiremock.com.google.common.base.Objects;
 
 import com.mercedesbenz.sechub.integrationtest.internal.TestJsonDeleteCount;
 
@@ -137,10 +136,10 @@ public class AssertAutoCleanupInspections {
         public ActionState validate(List<TestJsonDeleteCount> counts, StringBuilder problemMessageBuilder) {
             boolean found = false;
             for (TestJsonDeleteCount count : counts) {
-                if (!Objects.equal(variant, count.variant)) {
+                if (!Objects.equals(variant, count.variant)) {
                     continue;
                 }
-                if (!Objects.equal(className, count.className)) {
+                if (!Objects.equals(className, count.className)) {
                     continue;
                 }
                 if (expectedDeletes != count.deleteCount) {

